index.html

```diff
@@ -61,123 +72,100 @@ <h2 class="container job-title">I am a Software developer </h2>
+                        <li>bootsrap</li>
+                        <li>Ruby</li>
+                    </ul>
+                </div>
+                <button type="button" class="card-button" id="thirdTitle">See Project</button>
+            </div> 
+            <div class="work-card">
+                <div class="card-content flex-container ">
+                    <h4 class="title" >Profesional Art Printing Data</h4>
+                    <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
+                    <ul>
+                        <li>html</li>
+                        <li>bootsrap</li>
+                        <li>Ruby</li>
+                    </ul>
+                </div>
+                <button type="button" class="card-button" id="fourthTitle">See Project</button>
+            </div> 
+            <div class="work-card">
+                <div class="card-content  flex-container">
+                    <h4 class="title" >Profesional Art Printing Data</h4>
+                    <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
+                    <ul>
+                        <li>html</li>
+                        <li>bootsrap</li>
+                        <li>Ruby</li>
+                    </ul>
+                </div>
+                <button type="button" class="card-button" id="fifthTitle">See Project</button>
+            </div> 
+          <div class="work-card">
+                <div class="card-content flex-container">
+                    <h4 class="title" >Profesional Art Printing Data</h4>
+                    <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
+                    <ul>
+                        <li>html</li>
+                        <li>bootsrap</li>
+                        <li>Ruby</li>
+                    </ul>
+                </div>
+                <button type="button" class="card-button" id="sixthTitle">See Project</button>
+            </div> 
         </div>
-    </div>
-
-<!-- Sixth-Profesional-Art-Printing-Data -->
-<div class="card-section">
-    <div class="work-card">
-        <div class="card-content flex-container">
-            <h4 class="title">Profesional Art Printing Data</h4>
-            <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
-            <ul>
-                <li class="html">html</li>
-                <li class="Boot">Bootstrap</li>
-                <li class="Ruby">Ruby</li>
-            </ul>
-        </div>
-            <button class="project-button" type="button" id="sixthTitle">See Project</button>
-        </div>
-    </div>
-</section><!-- #Recent Works -->
+    </section>
 <!-- ======================
     About me
     ====================== -->
```

Comment:
On the original about me section the line occupies the entire width of the screen, so please make sure to fix it.

![image](https://user-images.githubusercontent.com/34779928/193109800-432b26c4-ab7c-4149-a71a-3713ceccad5b.png)